cognee/api/v1/search/search_v2.py

```
@@ -0,0 +1,57 @@
+        raise PermissionError("No user found in the system. Please create a user.")
+
+    own_document_ids = await get_document_ids_for_user(user.id)
+    search_results = await specific_search(search_type, query, user)
+
+    from uuid import UUID
+
+    filtered_search_results = []
+
+    for search_result in search_results:
+        document_id = search_result["document_id"] if "document_id" in search_result else None
+        document_id = UUID(document_id) if type(document_id) == str else document_id
```

Ruff

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)